### PR TITLE
TRAN-78: Add possibility to remove already existing records

### PR DESCRIPTION
**Pull request by sniedzielski**
TICKET: https://sd-transit.atlassian.net/browse/TRAN-78

Peek 2022-09-05 17-04

### CODE REVIEW

File: src/pages/menuUpload/CustomerTypePage.tsx [hunk 1]
```diff
@@ -105,18 +146,47 @@ function CustomerTypePage() {
...
+            mode="Delete"
           />,
         ]}
       />
       {customerTypes === undefined ? (
-        <Table columns={columns} data={[{ }]} editAction={toggleEditModal}>
+        <Table
+          columns={columns}
+          data={[{ }]}
+          editAction={toggleEditModal}
```
Comment (KarolinaKopacz) [inline src/pages/menuUpload/CustomerTypePage.tsx]: Here Table is empty, right? So `editAction` and `deleteAction` props I think are unnecessary ;)

File: src/redux/sagas/customerTypeSaga.tsx [hunk 2]
```diff
@@ -57,6 +58,20 @@ const putCustomerType = async (payload: ICustomerType, id: number) => {
   return data;
 };
 
+const deleteCustomerType = async (id: number) => {
+  const accessToken = JSON.parse(localStorage.getItem(sessionToken) as string).access;
+  const { data } = await apiClient.delete(
+    `customer_type/${id}/`,
```
Comment (KarolinaKopacz) [inline src/redux/sagas/customerTypeSaga.tsx]: The string `customer_type/` is using several time, so maybe better to use it above the code 
`const costometTypeUrl = 'customer_type/' ` :)

**Review by KarolinaKopacz — CHANGES_REQUESTED** [review]

File: src/redux/sagas/customerTypeSaga.tsx (hunk 2 above)
Comment (sniedzielski) [reply src/redux/sagas/customerTypeSaga.tsx]: ok, good catch. I will fix this.

File: src/pages/menuUpload/CustomerTypePage.tsx (hunk 1 above)
Comment (sniedzielski) [reply src/pages/menuUpload/CustomerTypePage.tsx]: ok.

**Comment by sniedzielski** [comment]
@KarolinaKopacz fixed!

**Review by KarolinaKopacz — APPROVED** [review]

**Pull request merged**